src/solver/optimisation/opt_gestion_des_bornes_cas_lineaire.cpp

```diff
@@ -177,10 +230,37 @@ void OPT_InitialiserLesBornesDesVariablesDuProblemeLineaire(PROBLEME_HEBDO* Prob
             Var = CorrespondanceVarNativesVarOptim->NumeroDeVariableDeLInterconnexion[Interco];
             CoutDeTransport = ProblemeHebdo->CoutDeTransport[Interco];
 
-            Xmax[Var] = ValeursDeNTC->ValeurDeNTCOrigineVersExtremite[Interco];
-
-            Xmin[Var] = -(ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco]);
+            StartNodeAdequacyPatchType = ProblemeHebdo->StartAreaAdequacyPatchType[Interco];
+            EndNodeAdequacyPatchType = ProblemeHebdo->EndAreaAdequacyPatchType[Interco];
+            SetToZeroLinkNTCForAdequacyPatchFirstStep
+              = SetNTCForAdequacyFirstStep(AdequacyFirstStep,
+                                           StartNodeAdequacyPatchType,
+                                           EndNodeAdequacyPatchType,
+                                           SetToZero12LinksForAdequacyPatch,
+                                           SetToZero11LinksForAdequacyPatch);
 
+            if (SetToZeroLinkNTCForAdequacyPatchFirstStep == Data::setToZero)
+            {
+                Xmax[Var] = 0.;
+                Xmin[Var] = 0.;
+            }
+            else if (SetToZeroLinkNTCForAdequacyPatchFirstStep == Data::setStartEndToZero)
+            {
+                Xmax[Var] = 0.;
+                Xmin[Var] = ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco] > ADEQUACY_MAX
+                              ? -ADEQUACY_MAX
+                              : -(ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco]);
```

Comment:
This should maybe be discussed with RTE. When managing/altering the NTC constraints first step of adequacy patch, depending on the options, new types of constraint are possible [0 +inf] and [-inf 0]. The second one creates problem to the solver and Antares provides no results. All other constraints [-value1 +value2] (value can be zero) and [-inf +inf] works fine in test cases.

Reply:
OK. Let's report to RTE.